### DISCUSSION TITLE
X e Y por milimetros

### DISCUSSION THREAD

**Discussion by GOGYNAVA** [arkypita/LaserGRBL#1003] [Q&A]
Hola alguien tiene algun boto para mover los ejes manualmente por mm igual que estan los botones del eje z?
Se agradeceria bastante. Un saludo

**Comment by StuartB4** [comment]
Esto funciona para el eje X, simplemente haga otro para el eje Y.

![yuiyiuyiu](https://user-images.githubusercontent.com/16980026/81869769-8a72b000-956c-11ea-89c7-fbc70616c42b.jpg)

**Comment by GOGYNAVA** [comment]
Muchas gracias. Mañana lo pruebo

Obtener Outlook para Android<https://aka.ms/ghei36>

________________________________
